apps/src/code-studio/components/progress/MiniView.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import CourseProgress from './CourseProgress';
+
+/**
+ * The course progress dropdown you get when you click the arrow in the header.
+ */
+const MiniView = React.createClass({
+  propTypes: {
+    hasFullProgress: PropTypes.bool.isRequired
+  },
+
+  render() {
+    const { hasFullProgress } = this.props;
+
+    if (!hasFullProgress) {
+      // Ideally we would specify inline CSS instead of using a classname here,
+      // but the image used here gets digested by rails, and we don't know the
+      // digested path
+      return <div className="loading"/>;
```

Comment:
@joshlory recently configured webpack to be able to [locate and digest dashboard assets itself](https://github.com/code-dot-org/code-dot-org/blame/ed858a64b5b1eac96cea52041df40e4edf3181da/apps/webpack.js#L48-L48), which lets us (sort of) share some assets between dashboard and apps.  Take a look at https://github.com/code-dot-org/code-dot-org/pull/13433/files for how he used this.